src/test/regress/output/appendonly.source

```diff
@@ -1714,3 +1714,50 @@ select count(*) from fix_ao_truncate_last_sequence;
 (1 row)
 
 abort;
+-- check reporting of missing attributes for AO table
+-- start_ignore
+drop table if exists test_reporting;
```

Comment:
ignore content in output is redundant

Reply:
This is a moment when 2 reviewers are contradicting. https://github.com/arenadata/gpdb/pull/671#discussion_r1477646355

Reply:
no! I mean you should delete contents between start and end ignore block in output

Reply:
Okay, although there are only a few examples of empty content between start and end. The vast majority of such a cases has something in between.

Reply:
> something in between

is redundant in output

Reply:
> is redundant in output